packages/react-core/src/deprecated/components/Wizard/examples/Wizard.md

```diff
@@ -375,5 +375,5 @@ class ProgressiveWizard extends React.Component {
 
 ### Wizard with drawer
 
-```ts file="./WizardWithDrawer.tsx"
+```js file="./WizardWithDrawer.tsx"
```

Comment:
Had to change this from being ts tagged to js tagged, having it ts tagged caused the build to fail.